packages/shared/lib/contexts/onboarding/index.ts

```diff
@@ -6,3 +6,4 @@ export * from './helpers'
 export * from './interfaces'
 export * from './stores'
 export * from './types'
+export * from './utils'
```

Comment:
p.s. we should export the helpers from the module

Reply:
Do you mean shouldn't?